Library/Homebrew/cmd/install.rb

```diff
@@ -142,7 +145,12 @@ def check_cellar
   def perform_preinstall_checks
     check_ppc
     check_writable_install_location
-    check_xcode
+    if MacOS::Xcode.installed?
+      check_xcode
+    else
+      opoo "You have not installed Xcode."
+      puts "Bottles may install correctly, but builds will fail!"
```

Comment:
Not all builds will fail. It'd be good to decide how to fail here rather than saying "this may fail" and then allowing it to happen?